src/plugins/piral-lazy/src/create.test.ts

```diff
@@ -22,13 +25,22 @@ function createMockContainer() {
   };
 }
 
+const mount = jest.fn();
```

Comment:
I'd recommend moving these two lines into the test.

Just to have it locally (otherwise, if you re-use `mount` it would already have been called from one test, which makes it quite difficult / unreliable to work).

Reply:
I moved the 2 lines - the results are still the same.

BTW you can review the piral-translate as well.

Reply:
It's not about that the results change now, but rather that having it not done well in the first place could / will lead to issues later (if not used carefully).